api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -284,7 +284,8 @@ message CommonResponse {
 
   // Instructions on how to manipulate the headers. When responding to an
   // HttpBody request, header mutations will only take effect if
-  // the current processing mode for the body is BUFFERED.
+  // the current processing mode for the body is: 1) BUFFERED; 2) or STREAMED and
+  // the filter configuration ``send_body_before_recv_header_resp``  is enabled.
```

Comment:
This should have a reference link, and should not be visible until the other field is implemented.
As the other field is no-implemented-hide, it should also not appear in the docs.
I think you can use a [#comment tag](https://github.com/envoyproxy/envoy/blob/bc3f240f54c161448df96644b8a87b3f4404143d/api/CONTRIBUTING.md?plain=1#L67) to hide this for now.

Reply:
done